Tests/FlowMagic-SDKTests/FlowMagic_SDKTests.swift

```diff
@@ -1,11 +1,11 @@
 import XCTest
-@testable import FlowMagicSDK
+@testable import FlowMagic_SDK
```

Comment:
Just to make sure, this is correct?  With an underscore?

Reply:
Yes, this is correct. In Swift, a hyphen ("-") in module names is replaced with an underscore ("_") because hyphens are not syntactically valid.